src/Nethermind/Nethermind.Consensus/Validators/TxValidator.cs

```diff
@@ -38,7 +38,14 @@ public bool IsWellFormed(Transaction transaction, IReleaseSpec releaseSpec)
                       while for an init it will be empty */
                    ValidateSignature(transaction.Signature, releaseSpec) &&
                    ValidateChainId(transaction) &&
-                   Validate1559GasFields(transaction, releaseSpec);
+                   Validate1559GasFields(transaction, releaseSpec) &&
+                   Validate3860Rules(transaction, releaseSpec);
+        }
+
+        private bool Validate3860Rules(Transaction transaction, IReleaseSpec releaseSpec)
+        {
+            bool aboveInitCode = transaction.IsContractCreation && releaseSpec.IsEip3860Enabled && (transaction.Data?.Length ?? 0) > releaseSpec.MaxInitCodeSize;
```

Comment:
`(transaction.Data?.Length ?? 0)` this is in few places, can we introduce on `Transaction` object property:

`DataLength => Data?.Length ?? 0` to avoid this check in multiple places?